Week-6/assignment-3/index.js

```diff
@@ -0,0 +1,30 @@
+const sort = (nums) => {
+  let zero = 0;
+  let one = 0;
+  let high = nums.length - 1;
+  let result = [];
+
+  for (let i = 0; i < nums.length; i++) {
+    if (nums[i] === 0) {
```

Comment:
I didnt understand the logic and naming of vars. Can u add comments to explaint?